recipes-asteroid/asteroid-sounds/asteroid-sounds_git.bb

```diff
@@ -0,0 +1,17 @@
+SUMMARY = "Default set of sounds for AsteroidOS under CC0"
+HOMEPAGE = "https://github.com/AsteroidOS/asteroid-sounds"
+LICENSE = "CC0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=2c1c00f9d3ed9e24fa69b932b7e7aff2"
+
+SRC_URI = "git://github.com/AsteroidOS/asteroid-sounds;protocol=https;branch=master"
```

Comment:
There's an error with this url.
The repo should be `git://github.com/AsteroidOS/asteroid-sounds.git`
And the `branch` should be `main` resulting in:
```
SRC_URI = "git://github.com/AsteroidOS/asteroid-sounds.git;protocol=https;branch=main"
```